libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -147,6 +148,10 @@ const AP_Param::GroupInfo AP_MotorsHeli::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("COL_LAND_MIN", 32, AP_MotorsHeli, _collective_land_min_deg, AP_MOTORS_HELI_COLLECTIVE_LAND_MIN),
 
+    // @Group: RSC_AROT_
+    // @Path: ../AP_Motors/AP_MotorsHeli_RSC.cpp
```

Comment:
```suggestion
    // @Path: ../AC_Autorotation/RSC_Autorotation.cpp
```
This should fix the test that isn't passing CI